byron/ledger/formal-spec/properties.tex

```diff
@@ -156,3 +156,76 @@ \subsection{UTxO Properties}
   $$
 
 \end{property}
+
+\subsection{PVBUMP STS Properties}
+\label{sec:pvbump-properties}
+
+Property~\ref{prop:pvbump-no-proposals} states that when no proposals are
+available, the system remains in the same state it started with.
+
+\begin{property}[PVBUMP on no proposals]\label{prop:pvbump-no-proposals}
```

Comment:
Maybe we need to find a better label for the property. What about "no change on empty future adoptions" or something like that ...

Reply:
Fixed!